src/NUnitTestExtractor/NUnitTestExtractor/Program.cs

```diff
@@ -44,55 +43,56 @@ static void Main(string[] args)
                         try
                         {
                             level = (Level)Enum.Parse(typeof(Level), _options.Level, true);
-                            WriteToTxtFile("file", _options.Output, _options.DLLs, level);
+                            WriteTestDllAndName("file", _options.Output, _options.DLLs, level);
                         }
                         catch (ArgumentException e)
                         {
-                            Console.WriteLine("ArgumentException: Please enter either Namespace, Class, or Function for level");
+                            Console.Error.WriteLine(e.GetType() + ": Please enter either Namespace, Class, or Function for level");
+                            Environment.Exit(-1);
                         }
                     }
                 }
             }
         }
-        //While reflection is not implemented this method writes path of dll if it exists with specified level to txt file
-        private static void WriteToTxtFile(string name, string outputDirectory, IList<string> dllDirectories, Level level)
+        /// <summary>
+        /// While reflection is not implemented method writes each dll verified to be existing to either a txt file or stdout along with the specified level
+        /// </summary>
+        /// <param name="name"> the name of the txt file that data is being appended to </param>
+        /// <param name="outputDirectory"> the directory in that the file is being saved to </param>
+        /// <param name="dllDirectories"> list of all dll directories which the user wishes to append to a file or stdout </param>
+        /// <param name="level"> Specifies the level of granuality to use for the output tests entered by user </param>
+        private static void WriteTestDllAndName(string name, string outputDirectory, IList<string> dllDirectories, Level level)
         {
             //user entered some dlls
             if (dllDirectories.Count != 0)
             {
+                StreamWriter writer;
```

Comment:
I would pull this and the if/else below it outside of this function and put it in the main() function.  Then instead of passing name and outputDirectory to this function, you can just pass the StreamWriter.
This would also make the function easier to unit test later since we can pass any type of StreamWriter to it, such as a StringWriter...